### ISSUE TITLE
Location of Model Output Files

### ISSUE THREAD

**Issue by michelleannesimon** [USEPA/SWMM-EPANET_User_Interface#348]
The program does not ask to save your work upon exiting.  We need to set up the output environment more cleanly.  We should be able to close a project but not exit.

the user should be able to set up an output environment
It makes a folder with ini file C:\Users\MSIMON\SWMMUI\Testing\test202002\test.ini\EPA\swwm.ini but does not routinely place outputs, txt,  and reports in same folder as input file or whatever the user specifies.

**Comment by michelleannesimon** [comment]
I have exited a few projects but the GUI does not save or ask me to save or to discard.

**Comment by PaulDudaRESPEC** [comment]
I retested the prompt to save upon exiting, and I found it to work reliably with the exception of closing using the upper right Red 'X'.  The commit referenced above adds the warning to save or discard when closing using the Red 'X'.

I've not been able to reproduce the reported issue with the model output files being written anywhere other than wherever the input file resides.  A specific example might be helpful, and it might be related to the restricted government computers.
